libs/SalesforceNetwork/SalesforceNetwork/SalesforceNetwork/Classes/Network/Action/CSFAction.m

```diff
@@ -60,10 +60,9 @@ + (NSURL*)urlForAction:(CSFAction*)action error:(NSError**)error {
     if (!action) {
         return nil;
     }
-    
-    NSMutableString *baseUrlString = [NSMutableString stringWithString:[action.enqueuedNetwork.account.credentials.apiUrl absoluteString]];
+    NSMutableString *baseUrlString = [NSMutableString stringWithString:[action.baseURL absoluteString]];
```

Comment:
The basic bug - this factory method shouldn't be married to ```Salesforce``` - it should rely on the ```action``` passed in for ```baseURL```.